.github/actions/run-integration-tests/action.yml

```diff
@@ -40,14 +40,14 @@ runs:
       run: |
         PGPASSWORD=asgthunder psql -h localhost -p 5432 -U asgthunder -d thunderdb -c "CREATE DATABASE identitydb;"
         PGPASSWORD=asgthunder psql -h localhost -p 5432 -U asgthunder -d thunderdb -c "CREATE DATABASE runtimedb;"
-        PGPASSWORD=asgthunder psql -h localhost -p 5432 -U asgthunder -d identitydb < backend/dbscripts/thunderdb/postgres.sql
-        PGPASSWORD=asgthunder psql -h localhost -p 5432 -U asgthunder -d runtimedb < backend/dbscripts/runtimedb/postgres.sql
+        PGPASSWORD=asgthunder psql -h localhost -p 5432 -U asgthunder -d identitydb < tests/integration/resources/dbscripts/thunderdb/postgres.sql
+        PGPASSWORD=asgthunder psql -h localhost -p 5432 -U asgthunder -d runtimedb < tests/integration/resources/dbscripts/runtimedb/postgres.sql
```

Comment:
Any specific reason to maintain separate file for the integration tests?  can't we reuse the `backend/dbscripts/thunderdb/postgres.sql` ?

With this developers have to update the schema in two places which might lead to mistakes

Reply:
We have some queries for pre created test resources in that directory (Ex: notification sender id/ idp id's needs to be updated in flow graphs). This requires the following steps.

1. Start the server
2. Create resources (notification senders/ IDP, etc)
3. Replace flow graphs
4. Restart the server and continue integration tests

Hence currently we're following the above approach. Moved all the files there to fix the current tests.
We should be able to get rid of this once the flow configuration API is available.

Reply:
Modifying above action only fixes the workflow runs, however same approach needs to be applied to the local integration tests execution as well..
This needs some refactoring. Shall we check this separately as not related to $subject of this PR?